benchmark/pom.xml

```diff
@@ -34,26 +28,10 @@
 			</plugin>
 			<plugin>
 				<artifactId>maven-assembly-plugin</artifactId>
-				<configuration>
```

Comment:
Why was the assembly configuration removed?
How do you create a release of the Benchmark now?

Reply:
You are right, this was a mistake. I restored the original config.